spec/features/station/user_can_view_a_station_spec.rb

```diff
@@ -0,0 +1,22 @@
+require_relative '../../spec_helper'
+
+RSpec.describe "when user visits one station path" do
+  it "user can see a station" do
+    station = Station.create(name: "Amara Station", dock_count: 4, city: "Denver", installation_date: "2014-06-07")
+
+    visit("/stations/#{station.id}")
+
+    expect(page).to have_content("Amara Station")
```

Comment:
again try to use `within` to scope where you're looking for this content. The within block adds to the robustness of a tests by asserting what the HTML structure looks like, not just what words are on the page.